src/core/src/bootstrap/ConfigurationFactory.py

```diff
@@ -240,4 +253,31 @@ def new_test_configuration(self, package_manager_name, package_manager_component
         configuration['config_env'] = Constants.TEST
         # perform desired modifications to configuration
         return configuration
+
+    def get_lifecycle_manager_component(self,vm_context):
+        """ fnding life cycle manager based on vm and returning component name add in the prod configuration"""
+        azure_package_manager_component = LifecycleManagerAzure
```

Comment:
variable naming for clarity (applies elsewhere too) -- azure_lifecycle_manager_component, etc